enumero/pureconfig/src/main/scala/io.buildo.enumero/PureConfigSupport.scala

```diff
@@ -0,0 +1,14 @@
+package io.buildo.enumero
+
+import pureconfig._
+import pureconfig.generic.auto._
+
+trait PureConfigSupport {
+  implicit def decodeCaseEnum[T <: CaseEnum](implicit s: CaseEnumSerialization[T]) =
+    ConfigReader[String].map(x => s.caseFromString(x).get)
+
+  implicit def decodeIndexedCaseEnum[T <: IndexedCaseEnum](implicit s: CaseEnumIndex[T]) =
+    ConfigReader[T#Index].map(x => s.caseFromIndex(x).get)
+}
+
+package object ppureconfig extends PureConfigSupport
```

Comment:
@gabro, I can't use `pureconfig` as the name for the package object as it's clashing with the pureconfig _library_. Any idea on how to solve this?

Reply:
Maybe leveraging `_root_` in imports would help to solve this.